packages/fe-fpm-writer/templates/page/1.94/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+    "sap.ui5": {
+        "dependencies": {
+            "minUI5Version": "1.94.0"
```

Comment:
This overrides the minUI5Version of my application. Can we remove it from the template?
![image](https://user-images.githubusercontent.com/91968887/139871264-4b2ce06a-3035-4eda-b03b-704c3b615e6f.png)

Reply:
Oha ... that is wrong. I wouldn't remove it but it should only override if the current `minUI5Version` is smaller.

Reply:
Or we have a validation thats checks if the app version is correct and throws an exception when I call e.g. `generateCustomPage`?

Reply:
Actually, we have the optional input parameter allowing the caller to define the compatibility requirement for UI5, so we should not set this value, it is the responsibility of the consumer. Removed with https://github.com/SAP/open-ux-tools/pull/39/commits/b4bac19ab15e43a892eafaafbfdc15fc5e8d229d